birdhouse/components/weaver/images/component-diagram.svg

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Do not edit this file with editors other than diagrams.net -->
```

Comment:
Just to validate. The actual diff is just the newline? Not sure if it's better to ignore `.svg`.

Reply:
Yeah it looks like the only change is the newline...

Ok, so we should add the following files to the ignore regex:

- birdhouse/components/weaver/images/component-diagram.svg
- .bumpversion.cfg

Any other ones you can think of?